### PR TITLE
feat(mnq): add how tos and quickstart

### PR DESCRIPTION
**Pull request by RoRoJ**
Add how tos and quickstart for M&Q GA

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: serverless/messaging/how-to/create-manage-queues.mdx [hunk 1]
```diff
@@ -1,115 +1,106 @@
...
     - **Message retention period**: The length of time (in seconds) that messages are kept in the queue before being deleted. Setting a longer message retention period allows for a longer interval between a message being sent and it being received. The default value is 60 seconds. The value must be between 60 seconds and 1 290 600 seconds (14 days).
     - **Maximum message size**: The maximum size (in kilobytes) of a message that can be sent to the queue. Messages greater than the defined size will be rejected. The default value is 256 KB. The value must be between 1 KB and 256 KB.
     - **Content Based Deduplication**: This setting is only available for FIFO queues. Enable content based deduplication if the message body is guaranteed to be unique for each message. A unique hash value is generated from the body of each message, which is used as its deduplication ID. This avoids the need to set a deduplication ID when sending messages.
 
-8. Click **Create queue** to finish.
+7. Click **Create queue** to finish.
```
Comment (ofranc) [inline serverless/messaging/how-to/create-manage-queues.mdx:44]: j'ai l'impression que c'est le 6. ici non ?

File: serverless/messaging/how-to/deactivate-delete-messaging.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
+
+2. Click **NATS**, then click the NATS account you want to delete.
+
+3. In the bottom panel, click the **Delete NATS account** button.
+
+    A popup displays, reminding you that the action is irreversible, and any associated queued messages and credentials will be deleted.
+
+5. Type `DELETE` and click **Delete NATS account** to confirm.
```
Comment (ofranc) [inline serverless/messaging/how-to/deactivate-delete-messaging.mdx:49]: ici aussi step 4 ?

File: serverless/messaging/how-to/create-credentials.mdx [hunk 3]
```diff
@@ -41,23 +46,29 @@ Credentials for NATS namespaces are not granular: they necessarily give full rea
     <Message type="important">
       The generated credentials are displayed only once. Make sure to download and safely store them before closing the window.
     </Message>
-5. Click **Download** and save the file, before closing the window.
 
-    You are returned to your namespace's overview page. The name of your newly-generated credentials and their access level is displayed in the **Credentials** tab.
+6. Click **Download** and save the file, before closing the window.
+
+    You are returned to your NATs account's overview page. The name of your newly-generated credentials and their access level is displayed in the **Credentials** panem.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/create-credentials.mdx:52]:
```suggestion
    You are returned to your NATs account's overview page. The name of your newly-generated credentials and their access level is displayed in the **Credentials** panel.
```

**Review by ofranc — APPROVED** [review]

File: serverless/messaging/how-to/create-manage-queues.mdx [hunk 4]
```diff
@@ -1,115 +1,106 @@
...
-4. Click the green `+` button to create a new queue. The queue creation wizard displays.
-    <Lightbox src="scaleway-create-queue-wizard.webp" alt="" />
-
-5. Choose your **queue type**, from one of the following:
+3. Choose your **queue type**, from one of the following:
     - **Standard**: Standard queues ensure at-least-once delivery. The exact order of messages in the queue is not preserved.
     - **FIFO**: FIFO queues (**F**irst **In** **F**irst **O**ut) preserves the exact ordering of messages.
 
-6. Enter a **name** for your queue. The name must be no more than 80 characters, and consist only of alphanumeric characters, hyphens, and underscores. Alternatively, you can go with the auto-generated name suggested for you.
+4. Enter a **name** for your queue. The name must be no more than 80 characters, and consist only of alphanumeric characters, hyphens, and underscores. Alternatively, you can go with the auto-generated name suggested for you.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/create-manage-queues.mdx:36]:
```suggestion
4. Enter a **name** for your queue. The name must contain no more than 80 characters, and consist only of alphanumeric characters, hyphens, and underscores. Alternatively, you can go with the auto-generated name suggested for you.
```

File: serverless/messaging/how-to/create-manage-queues.mdx [hunk 5]
```diff
@@ -1,115 +1,106 @@
...
     - **Message retention period**: The length of time (in seconds) that messages are kept in the queue before being deleted. Setting a longer message retention period allows for a longer interval between a message being sent and it being received. The default value is 60 seconds. The value must be between 60 seconds and 1 290 600 seconds (14 days).
     - **Maximum message size**: The maximum size (in kilobytes) of a message that can be sent to the queue. Messages greater than the defined size will be rejected. The default value is 256 KB. The value must be between 1 KB and 256 KB.
     - **Content Based Deduplication**: This setting is only available for FIFO queues. Enable content based deduplication if the message body is guaranteed to be unique for each message. A unique hash value is generated from the body of each message, which is used as its deduplication ID. This avoids the need to set a deduplication ID when sending messages.
 
-8. Click **Create queue** to finish.
+7. Click **Create queue** to finish.
+
+You are returned to the **Queues** tab, where your newly-created queue now displays. You can view its URL here, for use in your API/CLI operations.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/create-manage-queues.mdx:46]: "Here" as in the "Queues" tab or should there be a link to where they will see the URL?

File: serverless/messaging/how-to/deactivate-delete-messaging.mdx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [activated SQS or SNS, or created a NATS account](/serverless/messaging/how-to/get-started/)
+</Message>
+
+## How to deactivate SNS or SQS
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/deactivate-delete-messaging.mdx:27]:
```suggestion
1. Click **Messaging** in the **Serverless** section of the Scaleway console side menu. The list of Messaging and Queuing protocols displays.
```

File: serverless/messaging/how-to/deactivate-delete-messaging.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+4. In the bottom panel, click the **Deactivate** button for the protocol.
+
+    A popup displays, reminding you that any existing SQS (or SNS) resources you have will be permanently deleted, and that you will not be able to create any new queues/topics or credentials unless you reactivate the protocol.
+
+5. Click **Deactivate SQS** (or SNS) to confirm.
+
+## How to delete a NATS account
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/deactivate-delete-messaging.mdx:41]:
```suggestion
1. Click **Messaging** in the **Serverless** section of the Scaleway console side menu. The list of Messaging and Queuing protocols displays.
```

File: serverless/messaging/how-to/get-started.mdx [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+</Message>
+
+## How to create a NATS account
+
+To start using a NATS message broker, you must first create a NATS account. This sets a scope for your NATS credentials, messages, queues and stream.
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The following screen displays:
```
Comment (nerda-codes) [inline serverless/messaging/how-to/get-started.mdx:33]:
```suggestion
1. Click **Messaging** in the **Serverless** section of the Scaleway console side menu. The following screen displays:
```

File: serverless/messaging/how-to/get-started.mdx [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+    You are directed to the listing of your NATS accounts, where your newly-created NATS account now appears.
+
+    <Message type="tip">
+    You can create more NATS accounts if you wish, by clicking on the `+ Create NATS account` button.
+    </Message>
+
+
+## How to activate SQS or SNS
+
+SQS and SNS are handled as two separate protocols. Each one needs to be activated before you can start creating credentials, queues, topics etc for this protocol. You can choose to activate one or both protocols as you like. You can deactivate each protocol at any time.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/get-started.mdx:59]:
```suggestion
SQS and SNS are handled as two separate protocols. Each one needs to be activated before you can start creating credentials, queues, topics, etc. for this protocol. You can choose to activate one or both protocols as you like. You can deactivate each protocol at any time.
```

File: serverless/messaging/how-to/get-started.mdx [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The following screen displays:
```
Comment (nerda-codes) [inline serverless/messaging/how-to/get-started.mdx:61]:
```suggestion
1. Click **Messaging** in the **Serverless** section of the Scaleway console side menu. The following screen displays:
```

File: serverless/messaging/how-to/manage-credentials.mdx [hunk 11]
```diff
@@ -1,54 +1,72 @@
...
-4. Click **Revoke credential** to confirm. 
+5. Click **Update credentials**.
 
-    The credentials are revoked and you are returned to your namespace's overview page.
+    The credentials are updated and you are returned to the protocol's **Credentials** listing.
+
+
+## How to revoke SQS or SNS credentials
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/manage-credentials.mdx:45]: same comment as above but feel free to ignore

File: serverless/messaging/how-to/manage-credentials.mdx [hunk 12]
```diff
@@ -1,54 +1,72 @@
...
+
+    A popup displays, warning you that this action is irreversible and the key will no longer be able to access your service.
+
+4. Click **Revoke credentials**.
+
+    The credentials are revoked and you are returned to the protocol's **Credentials** listing.
+
+## How to revoke NATS credentials
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/manage-credentials.mdx:59]: same as above

File: serverless/messaging/how-to/manage-credentials.mdx [hunk 13]
```diff
@@ -1,54 +1,72 @@
...
+
+2. Click **NATS** and select the NATS account you want to revoke credentials for.
+
+3. In the **Credentials** pabel, click the <Icon name="delete" /> button next to the credentials you want to revoke.
+
+    A popup displays, warning you that this action is irreversible and the key will no longer be able to access your service.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/manage-credentials.mdx:65]:
```suggestion
    A pop-up displays, warning you that this action is irreversible and the key will no longer be able to access your service.
```

File: serverless/messaging/how-to/deactivate-delete-messaging.mdx [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+2. Click **SQS** or **SNS** depending on which protocol you want to deactivate.
+
+3. Click the **Settings** tab.
+
+4. In the bottom panel, click the **Deactivate** button for the protocol.
+
+    A popup displays, reminding you that any existing SQS (or SNS) resources you have will be permanently deleted, and that you will not be able to create any new queues/topics or credentials unless you reactivate the protocol.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/deactivate-delete-messaging.mdx:35]:
```suggestion
    A pop-up displays, reminding you that any existing SQS (or SNS) resources you have will be permanently deleted, and that you will not be able to create any new queues/topics or credentials unless you reactivate the protocol.
```

File: serverless/messaging/how-to/deactivate-delete-messaging.mdx [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+
+## How to delete a NATS account
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
+
+2. Click **NATS**, then click the NATS account you want to delete.
+
+3. In the bottom panel, click the **Delete NATS account** button.
+
+    A popup displays, reminding you that the action is irreversible, and any associated queued messages and credentials will be deleted.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/deactivate-delete-messaging.mdx:47]:
```suggestion
    A pop-up displays, reminding you that the action is irreversible, and any associated queued messages and credentials will be deleted.
```

File: serverless/messaging/how-to/manage-credentials.mdx [hunk 16]
```diff
@@ -1,54 +1,72 @@
...
-4. Select **Update credentials**. A popup displays.
-    <Lightbox src="scaleway-update-credentials-popup.webp" alt="" />
-5. Make the required updates to your credentials. You can modify the name, and the permissions.
-6. Click **Update credentials**.
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
 
-    The credentials are updated and you are returned to the namespace's **Overview** page. The updated credentials can be seen in the **Credentials** tab.
+2. Click SQS or SNS depending on which service you want to update credentials for.
 
+3. In the **Credentials** tab, click the <Icon name="more" /> button next to the credentials you want to update, and select **Update credentials**. A popup displays, such as the one below for SQS:
```
Comment (nerda-codes) [inline serverless/messaging/how-to/manage-credentials.mdx:32]:
```suggestion
3. In the **Credentials** tab, click the <Icon name="more" /> button next to the credentials you want to update, and select **Update credentials**. A pop-up displays, such as the one below for SQS:
```

File: serverless/messaging/how-to/manage-credentials.mdx [hunk 17]
```diff
@@ -1,54 +1,72 @@
...
+
+## How to revoke SQS or SNS credentials
+
+1. Click **Messaging** in the **Serverless** section of the side menu. The list of Messaging and Queuing protocols displays.
+
+2. Click **SQS** or **SNS** depending on which service you want to revoke credentials for.
+
+3. In the **Credentials** tab, click the <Icon name="more" /> button next to the credentials you want to revoke and select **Revoke**.
+
+    A popup displays, warning you that this action is irreversible and the key will no longer be able to access your service.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/manage-credentials.mdx:51]:
```suggestion
    A pop-up displays, warning you that this action is irreversible and the key will no longer be able to access your service.
```

File: serverless/messaging/how-to/create-manage-queues.mdx (hunk 1 above)
Comment (RoRoJ) [reply serverless/messaging/how-to/create-manage-queues.mdx]: Fixed

File: serverless/messaging/how-to/deactivate-delete-messaging.mdx (hunk 2 above)
Comment (RoRoJ) [reply serverless/messaging/how-to/deactivate-delete-messaging.mdx]: Fixed

File: serverless/messaging/how-to/create-manage-queues.mdx (hunk 5 above)
Comment (RoRoJ) [reply serverless/messaging/how-to/create-manage-queues.mdx]: I've clarified it in the text

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**